client/asset/dcr/dcr.go

```diff
@@ -785,25 +786,26 @@ func (dcr *ExchangeWallet) feeRate(confTarget uint64) (uint64, error) {
 }
 
 // externalFeeEstimator gets the fee rate from the external API
-func externalFeeEstimator(nb uint64) (float64, error) {
-	url := externalApiUrl + "/utils/estimatefee?nbBlocks=" + strconv.Itoa(int(nb))
-	r, err := http.Get(url)
+func (dcr *ExchangeWallet) externalFeeEstimator(nb uint64) (float64, error) {
```

Comment:
Not a problem but I liked this as a plain function and it would be idiomatic like `externalFeeEstimator(ctx context.Context, nb uint64) (float64, error)`.